### DISCUSSION TITLE
Nice theme in gifs!

### DISCUSSION THREAD

**Discussion by boydkelly** [nvim-orgmode/orgmode#91] [General]
Hi,  Sorry for the noise here.   What is the status line/theme in your gifs?

https://github.com/kristijanhusak/orgmode.nvim#agenda

Thanks!!!!!

**Comment by kristijanhusak** [comment]
It's one-light from https://github.com/RRethy/nvim-base16.

**Reply by boydkelly** [reply]
What about the status bar?

**Reply by kristijanhusak** [reply]
It's custom, see https://github.com/kristijanhusak/neovim-config/blob/master/nvim/lua/partials/statusline.lua
